Arrowgene.Ddon.Shared/Entity/Structure/CDataMDataCraftGradeUpRecipe.cs

```diff
@@ -12,11 +12,11 @@ public CDataMDataCraftGradeupRecipe()
         }
         public uint RecipeID { get; set; }
         public uint ItemID { get; set; }
-        public uint Unk0 { get; set; } // This appears to be "NextGrade", but that might be stored in the DB? 
+        public uint Upgradable { get; set; } // 1 is yes, 0 is no. will prompt a warning "will become max grade" 
         public uint GradeupItemID { get; set; }
         public uint Cost { get; set; }
         public uint Exp { get; set; }
-        public bool Unk1 { get; set; } // This appears to determine if its the max version or not, This may also be stored in the DB though.
+        public bool Unk1 { get; set; } // Prompts a warning that you can't get multiple levels for this item.
```

Comment:
Can we give this a name like `PromptUpgradeWarning`?

Reply:
This is a bool. It only has two value. You can find a more human friendly name for it.